docs/init.md

```diff
@@ -91,4 +91,26 @@ module.exports = {
 };
 ```
 
+## Breaking change >6.3.1
```

Comment:
I suggest not calling it a breaking change as it won't break things, but rather documenting this in the following manner:

**>6.3.1**  the responsibility of showing the user progress of the "Executing post init script" goes to the implementor instead of being handled by cli. the `ora` package is used to display progress.